tests/foreman/cli/test_subnet.py

```diff
@@ -71,115 +71,85 @@ class SubnetTestCase(CLITestCase):
     :CaseImportance: High
     """
 
-    @tier1
-    def test_positive_create_with_name(self):
-        """Check if Subnet can be created with random names
-
-        :id: 99cda3eb-3912-461b-83bd-f906b78eeca0
-
-        :expectedresults: Subnet is created and has random name
-
-        :CaseImportance: Critical
-        """
-        for name in valid_data_list():
-            with self.subTest(name):
-                subnet = make_subnet({'name': name})
-                self.assertEqual(subnet['name'], name)
-
     @tier1
     @upgrade
-    def test_positive_create_with_address_pool(self):
-        """Create subnet with valid address pool
+    def test_positive_CRUD(self):
+        """Create, update and delete subnet
 
         :id: d74a52a7-df56-44ef-89a3-081c14e81e43
 
-        :expectedresults: Subnet is created and address pool is set
-
-        :CaseImportance: Critical
-        """
-        for pool in valid_addr_pools():
-            with self.subTest(pool):
-                pool.sort()
-                mask = '255.255.255.0'
-                # generate pool range from network address
-                network = gen_ipaddr()
-                from_ip = re.sub(r'\d+$', str(pool[0]), network)
-                to_ip = re.sub(r'\d+$', str(pool[1]), network)
-                subnet = make_subnet({
-                    u'from': from_ip,
-                    u'mask': mask,
-                    u'network': network,
-                    u'to': to_ip,
-                })
-                self.assertEqual(subnet['start-of-ip-range'], from_ip)
-                self.assertEqual(subnet['end-of-ip-range'], to_ip)
-
-    @tier1
-    def test_positive_create_with_domain(self):
-        """Check if subnet with domain can be created
-
-        :id: 7ce7b139-d2b7-44f4-9c1a-1bd591f95334
-
-        :expectedresults: Subnet is created and has new domain assigned
+        :expectedresults: Subnet is created, updated and deleted
 
         :CaseImportance: Critical
         """
-        domain = make_domain()
-        subnet = make_subnet({'domain-ids': domain['id']})
-        self.assertIn(domain['name'], subnet['domains'])
-
-    @tier1
-    def test_positive_create_with_domains(self):
-        """Check if subnet with different amount of domains can be
-        created in the system
-
-        :id: e81ddec5-38b0-4c42-b89b-5cf2af580d39
-
-        :expectedresults: Subnet is created and has new domains assigned
-        """
+        name = valid_data_list()[0]
+        pool = valid_addr_pools()[0]
+        pool.sort()
```

Comment:
You can just `pool = sorted(valid_addr_pools()[0])`.

Same applies to `pool.sort()` used below.